graph/pull.go

```diff
@@ -491,7 +491,7 @@ func (s *TagStore) pullV2Tag(eng *engine.Engine, r *registry.Session, out io.Wri
 
 				io.Copy(tmpFile, utils.ProgressReader(ioutil.NopCloser(tarSumReader), int(l), out, sf, false, utils.TruncateID(img.ID), "Downloading"))
 
-				out.Write(sf.FormatProgress(utils.TruncateID(img.ID), "Verifying Checksum", nil))
+				out.Write(sf.FormatProgress(utils.TruncateID(img.ID), "Verifying Checksum - this is a tech preview, don't rely on it for security yet.", nil))
```

Comment:
I think the only other place it belongs is next to the verified message.  Keep in mind the current verified message will only show up for official images signed by official image build key.  While we might change the checksum method, the fact that we are verifying the checksum using tarsum IMO does not need to need the preview subtext.